specification/azurefleet/resource-manager/Microsoft.AzureFleet/stable/2024-11-01/azurefleet.json

```diff
@@ -0,0 +1,3581 @@
+        },
+        "content": {
+          "type": "string",
+          "format": "password",
+          "description": "Specifies the XML formatted content that is added to the unattend.xml file for\nthe specified path and component. The XML must be less than 4KB and must\ninclude the root element for the setting or feature that is being inserted.",
+          "x-ms-mutability": [
+            "update",
+            "create"
+          ],
+          "x-ms-secret": true
+        }
+      }
+    },
+    "ApiEntityReference": {
+      "type": "object",
+      "description": "The API entity reference.",
+      "properties": {
+        "id": {
+          "type": "string",
+          "format": "arm-id",
+          "description": "The ARM resource id in the form of\n/subscriptions/{SubscriptionId}/resourceGroups/{ResourceGroupName}/..."
+        }
+      }
+    },
+    "ApiError": {
+      "type": "object",
+      "description": "ApiError for Fleet",
+      "properties": {
+        "code": {
+          "type": "string",
+          "description": "The error code."
+        },
+        "target": {
+          "type": "string",
+          "description": "The target of the particular error."
+        },
+        "message": {
+          "type": "string",
+          "description": "The error message."
+        },
+        "details": {
+          "type": "array",
+          "description": "The API error details",
+          "items": {
+            "$ref": "#/definitions/ApiErrorBase"
+          },
+          "x-ms-identifiers": [
+            "message",
+            "target"
+          ]
+        },
+        "innererror": {
+          "$ref": "#/definitions/InnerError",
+          "description": "The API inner error"
+        }
+      }
+    },
+    "ApiErrorBase": {
+      "type": "object",
+      "description": "API error base.",
+      "properties": {
+        "code": {
+          "type": "string",
+          "description": "The error code."
+        },
+        "target": {
+          "type": "string",
+          "description": "The target of the particular error."
+        },
+        "message": {
+          "type": "string",
+          "description": "The error message."
+        }
+      }
+    },
+    "ApplicationProfile": {
+      "type": "object",
+      "description": "Contains the list of gallery applications that should be made available to the\nVM/VMSS",
+      "properties": {
+        "galleryApplications": {
+          "type": "array",
+          "description": "Specifies the gallery applications that should be made available to the VM/VMSS",
+          "items": {
+            "$ref": "#/definitions/VMGalleryApplication"
+          },
+          "x-ms-identifiers": [
+            "packageReferenceId"
+          ]
+        }
+      }
+    },
+    "ArchitectureType": {
+      "type": "string",
+      "description": "Architecture types supported by Azure VMs.",
+      "enum": [
+        "ARM64",
+        "X86"
```

Comment:
This should be X64. Azure VMs are no longer supporting X86.

Reply:
Updated.